.watchmanconfig

```diff
@@ -1,6 +1,8 @@
 {
     "ignore_dirs": [
         "android/build",
-        "target"
+        "target",
+	".clj-kondo",
+	".shadow-cljs"
     ]
```

Comment:
while debugging metro, found that watchman is keeping track of these two directories (e.g. `.clj-kondo` has ~12000 files usually and `.shadow-cljs` ~3000). On top of that, working in the editor/IDE causes constant changes to `.clj-kondo`'s cache, which re-trigger watchman making it do extra work for no reason (and also hogging up my CPU at times).